WooCommerce/Classes/POS/Presentation/CartView.swift

```diff
@@ -1,10 +1,12 @@
 import SwiftUI
 
 struct CartView: View {
-    @ObservedObject private var viewModel: PointOfSaleDashboardViewModel
+    @ObservedObject private var dashboardViewModel: PointOfSaleDashboardViewModel
```

Comment:
The reference to the dashboard is currently necessary due to `.submitCart()`. This calls for `syncOrder(for cartProducts: [CartItem]) async` which contains `OrderService` and card reader logic